nautobot/utilities/tables.py

```diff
@@ -317,3 +339,26 @@ def render(self, value):
             trunc = Truncator(value)
             value = trunc.words(self.truncate_words)
         return value
+
+
+class CustomFieldColumn(tables.Column):
+    """
+    Display custom fields in the appropriate format.
+    """
+
+    def render(self, record, bound_column, value):
+        if isinstance(value, list):
+            if len(value):
+                template = ""
+                for v in value:
+                    template += f'<span class="label label-default">{v}</span> '
+            else:
+                template = '<span class="text-muted">&mdash;</span>'
```

Comment:
To address the issue with bool displays (and make the UI more consistent with most other Nautobot tables), I think you'd want to add the following here:

```python
        elif isinstance(value, bool):
            if value:
                template = '<span class="text-success"><i class="mdi mdi-check-bold"></i></span>'
            else:
                template = '<span class="text-danger"><i class="mdi mdi-close-thick"></i></span>'
```

Reply:
and indeed, that's a very easy fix! Should've read the change twice over.

Thanks for the pointer, pushing the change now. :)